addon-tools/report-creator/report/create_test.go

```diff
@@ -94,27 +91,11 @@ func updateCompareOutput(t *testing.T, test Test) {
 	require.NoError(t, cmpCmd.Flags().Set("recursive", "true"))
 	require.NoError(t, cmpCmd.Flags().Set("output", compare.Json))
 	cmpCmd.Run(cmpCmd, []string{})
-	if err := os.WriteFile(test.getJSONPath(), out.Bytes(), 0644); err != nil { // nolint:gocritic,gosec
-		t.Fatalf("test %s failed to update test file: %s", test.getJSONPath(), err)
-	}
-}
-
-func getGoldenValue(t *testing.T, fileName string, value []byte) {
-	if *update {
-		t.Log("update golden file")
-		if err := os.WriteFile(fileName, value, 0644); err != nil { // nolint:gocritic,gosec
-			t.Fatalf("test %s failed to update golden file: %s", fileName, err)
-		}
-	}
-	expected, err := os.ReadFile(fileName)
-	if err != nil {
-		t.Fatalf("test %s failed reading .golden file: %s", fileName, err)
-	}
-	require.Equal(t, string(expected), string(value))
+	result := testutils.GetFile(t, test.getJSONPath(), testutils.RemoveInconsistentInfo(t, out.String()), update)
+	require.Equal(t, result, testutils.RemoveInconsistentInfo(t, out.String()))
```

Comment:
I think these two should be here
259be8654dd520474eaeda133e5b9da4184f68c011f6ff51e40feb0515865aecR78